Bottom: 5e93d68409472371562e2cc7dd048f9952cc7282
Top:    2fbd40adc069aea64f45509151bd476440fc3c3d
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-21 14:27:32 +1300

Refresh of threadsafestreamresult

---

diff --git a/testtools/tests/test_testsuite.py b/testtools/tests/test_testsuite.py
index e466080..85988d1 100644
--- a/testtools/tests/test_testsuite.py
+++ b/testtools/tests/test_testsuite.py
@@ -165,7 +165,7 @@ class TestConcurrentStreamTestSuiteRun(TestCase):
 Traceback (most recent call last):
   File "...testtools/testsuite.py", line 188, in _run_test
     test.run(process_result)
-TypeError: run() takes exactly 1 ...argument (2 given)
+TypeError: run() takes ...1 ...argument (2 given)
 """, doctest.ELLIPSIS))
         events = [event[0:6] + (None,) for event in events]
         events[1] = events[1][:2] + (None,) + events[1][3:]
